plugins/rtmp-services/data/services.json

```diff
@@ -2846,6 +2846,139 @@
+                },
+                {
+                    "name": "Live Stream Server #06",
+                    "url": "rtmps://live-stream-server-06.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #07",
+                    "url": "rtmps://live-stream-server-07.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #08",
+                    "url":"rtmps://live-stream-server-08.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #09",
+                    "url": "rtmps://live-stream-server-09.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #10",
+                    "url": "rtmps://live-stream-server-10.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #11",
+                    "url": "rtmps://live-stream-server-11.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #12",
+                    "url": "rtmps://live-stream-server-12.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #13",
+                    "url": "rtmps://live-stream-server-13.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #14",
+                    "url": "rtmps://live-stream-server-14.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #15",
+                    "url": "rtmps://live-stream-server-15.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #16",
+                    "url": "rtmps://live-stream-server-16.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #17",
+                    "url": "rtmps://live-stream-server-17.wavezoo.com:8443/LiveApp"},
+                {
+                    "name": "Live Stream Server #18",
+                    "url": "rtmps://live-stream-server-18.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #19",
+                    "url": "rtmps://live-stream-server-19.wavezoo.com:8443/LiveApp"
+                },
+                {
+                    "name": "Live Stream Server #20",
+                    "url": "rtmps://live-stream-server-20.wavezoo.com:8443/LiveApp"
+                }
+            ],
+            "protocol": "RTMPS",
+            "supported video codecs": [
+                "h264",
+                "hevc"
+            ],
+            "supported audio codecs": [
+                "aac",
+                "opus"
```

Comment:
Unless your servers support [enhanced RTMP](https://github.com/veovera/enhanced-rtmp) do not specify "hevc" and "opus" here (or omit those properties entirely).